pkgs/development/web/nodejs/build-node-package.nix

```diff
@@ -295,9 +295,10 @@ let
     shellHook = ''
       ${preShellHook}
       export PATH=${nodejs}/bin:$(pwd)/node_modules/.bin:$PATH
-      mkdir -p node_modules
+      mkdir -p node_modules/.bin
       ${concatMapStrings (dep: ''
         ln -sfv ${dep}/lib/node_modules/${dep.pkgName} node_modules/
+        ln -sfv ${dep}/lib/node_modules/.bin/* node_modules/.bin/
```

Comment:
Looks like it will, but the hook won't be interrupted. I tried this on our project which is a mid-sized node project built with nix.

On the other hand, we changed a few things and now we don't depend on this commit. It can wait or be closed.